.vib/spring-cloud-dataflow/cypress/cypress/integration/spring-cloud-dataflow_spec.js

```diff
@@ -0,0 +1,108 @@
+/// <reference types="cypress" />
+import { random } from './utils.js';
+import { importAnApplication, createATask } from './prepare-app-state.js';
+
+it('allows getting Spring Cloud Dataflow info', () => {
+  cy.visit('/dashboard');
+  cy.get('.signpost-trigger').click();
+  cy.get('.signpost-content-body')
+    .invoke('text')
+    .should('match', /.*[0-9].*/);
+  cy.contains('button', 'More info').click();
+  cy.contains('.modal-content', 'Core')
+    .and('contain', 'Dashboard')
+    .and('contain', 'Shell');
+});
+
+it('allows a stream to be created and deployed', () => {
+  const STREAM = 'mongodb | cassandra';
+  const STREAM_APPLICATION = 'Stream application starters for Kafka/Maven';
+
+  importAnApplication(STREAM_APPLICATION);
+  cy.visit('/dashboard');
+  cy.get('[routerlink="streams/list"').click();
```

Comment:
As a go-to policy, do we want to access the page where the testing will take place by navigating the UI or using `cy.visit`? 

I don't really have any strong thoughts one way or the other, as I think both are error-prone, but having a defined policy here will be helpful to maintain a single style.

Reply:
We have the policy to use the URL on the navigation to avoid flakiness. 
I forgot about it for a time, hence the mistake!